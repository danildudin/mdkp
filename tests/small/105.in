3 3
51 54 27 
23 25 50 129
44 73 19 527
20 32 93 407
