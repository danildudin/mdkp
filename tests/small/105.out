132
3
0 1 2 
